### PR TITLE
Adding feature to change port status from Kilda GUI

### PR DESCRIPTION
**Pull request by neeraj890kumar**

### CODE REVIEW

File: services/src/openkilda-gui/src/main/java/org/openkilda/integration/model/PortConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.io.Serializable;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Getter @Setter @NoArgsConstructor @ToString
```
Comment (ashishaggarwalCS) [inline services/src/openkilda-gui/src/main/java/org/openkilda/integration/model/PortConfiguration.java]: Use Lombok Data

File: services/src/openkilda-gui/src/main/java/org/openkilda/integration/service/SwitchIntegrationService.java [hunk 2]
```diff
@@ -291,4 +294,29 @@ public String getSwitchRules(String switchId) {
...
+            HttpResponse response = restClientManager.invoke(
+                    applicationProperties.getConfigSwitchPort().replace("{switch_id}", switchId).replace("{port_no}",
+                            port),
+                    HttpMethod.PUT, objectMapper.writeValueAsString(configuration), "application/json",
+                    applicationService.getAuthHeader());
+            if (RestClientManager.isValidResponse(response)) {
+                return restClientManager.getResponse(response, ConfiguredPort.class);
+            }
+        } catch (JsonProcessingException e) {
+            LOGGER.error("Inside configurePort  Exception :", e);
```
Comment (gauravchug) [inline services/src/openkilda-gui/src/main/java/org/openkilda/integration/service/SwitchIntegrationService.java]: Provide relevant exception logs.

File: services/src/openkilda-gui/src/main/resources/ui/js/ports/portobject.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		}).fail(function(error) {
+			$('#port_detail_loading').hide();
+			common.infoMessage(error.responseJSON['error-auxiliary-message'],'error');
+		})
+	}
+	
+	cancelConfigurePort () {
+		$('.port_details_div_status').show();
+		$('#edit_port_status').hide();
+		$('#save_configure_port').hide();
```
[the remark this answers is not in the document]
Comment (neeraj890kumar) [reply services/src/openkilda-gui/src/main/resources/ui/js/ports/portobject.js]: updated

File: services/src/openkilda-gui/src/main/java/org/openkilda/utility/ApplicationProperties.java [hunk 4]
```diff
@@ -134,4 +137,8 @@ public String getUpdateFlow() {
     public String getFlowPath() {
         return flowPath;
     }
+
+    public String getConfigSwitchPort() {
```
[the remark this answers is not in the document]
Comment (neeraj890kumar) [reply services/src/openkilda-gui/src/main/java/org/openkilda/utility/ApplicationProperties.java]: updated

File: services/src/openkilda-gui/src/main/java/org/openkilda/integration/model/response/ConfiguredPort.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Getter @Setter @NoArgsConstructor @ToString
```
[the remark this answers is not in the document]
Comment (neeraj890kumar) [reply services/src/openkilda-gui/src/main/java/org/openkilda/integration/model/response/ConfiguredPort.java]: updated

File: services/src/openkilda-gui/src/main/java/org/openkilda/integration/service/SwitchIntegrationService.java (hunk 2 above)
Comment (neeraj890kumar) [reply services/src/openkilda-gui/src/main/java/org/openkilda/integration/service/SwitchIntegrationService.java]: updated

File: services/src/openkilda-gui/src/main/java/org/openkilda/integration/model/PortConfiguration.java (hunk 1 above)
Comment (neeraj890kumar) [reply services/src/openkilda-gui/src/main/java/org/openkilda/integration/model/PortConfiguration.java]: updated

**Pull request merged**